src/components/Inventory/withdraw/BatchWithdrawFlow.tsx

```diff
@@ -63,7 +71,8 @@ export default function BatchWithdrawFlow(props: BatchWithdrawFlowProps): JSX.El
     populateBatches();
   }, [batchIds]);
 
-  const onPurposeSelected = () => {
+  const onWithdrawalConfigured = (withdrawal: NurseryWithdrawal) => {
```

Comment:
updated this to take in the new value of withdrawal with filtered down batches.
ideally we don't want to be passing setStates across components, each component should ideally support an onChangexyz and allow the parent to control its own state.